src/ui/pages/Backup/VerifyBackup.tsx

```diff
@@ -127,13 +143,13 @@ export function VerifyBackup({ onSuccess }: { onSuccess(): void }) {
               >
                 Verify
               </Button>
-              {validationError ? (
+              {verifyMutation.error ? (
                 <UIText
                   kind="caption/regular"
                   color="var(--negative-500)"
                   style={errorStyle}
                 >
-                  Incorrect seed phrase
+                  {(verifyMutation.error as Error).message || 'unknown error'}
```

Comment:
We have an awesome helper precisely for such cases 🙂:
```suggestion
                  {getError(verifyMutation.error).message}
```